bindings/wasm/lib/bindings.ts

```diff
@@ -20,6 +21,15 @@ const callUtilsMethod = (method: __UtilsMethods__): any => {
     }
 };
 
+const getSecretManagerFromWallet = (
+    method: WalletMethodHandler,
+): SecretManagerMethodHandler => {
+    // TODO figure out why this one is extensible but client isnt
```

Comment:
Is this still the case? Since we switched to napi this might have changed something

Reply:
So actually wallet.getSecretManager is completely broken in wasm, as there is no replacement for the Neon External type.
Id like to make a new issue for that, so we can merge this anytime soon

Reply:
Then open one? :trollface: